Makefile

```diff
@@ -184,10 +184,19 @@ test-cmd: build
 # Example:
 #   make test-end-to-end
 test-end-to-end:
+    # remove as soon as the CI is updated to call test-end-to-end-unit
 	COVERAGE_SPEC=' ' DETECT_RACES='false' TIMEOUT='10m' hack/test-go.sh ./test/end-to-end
 	hack/test-end-to-end.sh
 .PHONY: test-end-to-end
 
+# Run end to end tests. Uses whatever binaries are currently built.
+#
+# Example:
+#   test-end-to-end-unit
+test-end-to-end-unit:
+	COVERAGE_SPEC=' ' DETECT_RACES='false' TIMEOUT='10m' hack/test-go.sh ./test/end-to-end
```

Comment:
Again we _could_ do this but those tests in `test/end-to-end/router_test.go` need direct access to the Docker socket so this is a non-starter